### PR TITLE
Upgrade testNG to 6.11, use force dependency resolution for testNG

### PR DESCRIPTION
**Pull request by cmnbroad**
And remove unnecessary dependencies introduced for GenomicsDB.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -58,6 +58,7 @@ final htsjdkVersion = System.getProperty('htsjdk.version','2.9.1')
 final sparkVersion = System.getProperty('spark.version', '2.0.2')
 final hadoopBamVersion = System.getProperty('hadoopBam.version','7.8.0')
 final genomicsdbVersion = System.getProperty('genomicsdb.version','0.5.0-proto-3.0.0-beta-1')
+final testNGVersion = 'org.testng:testng:6.11'
```
Comment (droazen) [inline build.gradle]: Should be just the version, not the full artifact ID

File: build.gradle [hunk 2]
```diff
@@ -74,6 +75,13 @@ configurations.all {
     all*.exclude group: 'com.google.guava', module: 'guava-jdk5'
 }
 
+configurations.compile {
+    resolutionStrategy {
+        // force testng dependency so we don't pick up a different version via GenomicsDB
+        force testNGVersion
+    }
+}
```
Comment (droazen) [inline build.gradle]: Move to `configurations.all`

**Comment by droazen** [comment]
Merge after addressing comments

File: build.gradle (hunk 1 above)
Comment (lbergelson) [inline build.gradle]: change to just the version here

File: build.gradle [hunk 3]
```diff
@@ -74,6 +75,13 @@ configurations.all {
     all*.exclude group: 'com.google.guava', module: 'guava-jdk5'
 }
 
+configurations.compile {
+    resolutionStrategy {
```
Comment (lbergelson) [inline build.gradle]: I would move this into the configurations.all block

**Pull request merged**